crates/mongo-wasmtime/src/bindings.rs

```diff
@@ -0,0 +1,329 @@
+pub mod wasi_mongo {
```

Comment:
The bindings file doesn't have to be committed.

Reply:
I will remove it before the final commit :). Makes it easier for me visually look at the generated functions and types.